osu.Game/Overlays/ChatOverlay.cs

```diff
@@ -203,10 +207,21 @@ protected override bool OnDragStart(InputState state)
 
         protected override bool OnDrag(InputState state)
         {
-            Trace.Assert(state.Mouse.PositionMouseDown != null);
+            if (isDragging)
+            {
+                Trace.Assert(state.Mouse.PositionMouseDown != null);
+
+                chatHeight.Value = startDragChatHeight - (state.Mouse.Position.Y - state.Mouse.PositionMouseDown.Value.Y) / Parent.DrawSize.Y;
+                return base.OnDrag(state);
+            }
 
-            chatHeight.Value = startDragChatHeight - (state.Mouse.Position.Y - state.Mouse.PositionMouseDown.Value.Y) / Parent.DrawSize.Y;
-            return base.OnDrag(state);
+            return true;

```

Comment:
i mean since we only got here after returning turn to `OnDragStart` we can probably just always return `true` here, outside the `if`